roles/openshift_health_checker/test/docker_image_availability_test.py

```diff
@@ -115,6 +115,39 @@ def execute_module(module_name=None, *_):
     assert "required Docker images are not available" in actual['msg']
 
 
+def test_no_known_registries():
```

Comment:
👍 for adding more tests! It does seem a bit out of place in this commit but I wouldn't block on it.

Reply:
Had to add it because coverage % went down from removing branches that were covered.  Needed coverage to not go down to pass.

Reply:
FYI, coverage tests are non-blocking, but big :+1: for keeping coverage up.